docs/how-to/configure.md

```diff
@@ -85,11 +85,16 @@ As well as via the *PMM Settings* page, you can also disable telemetry with the
 > - If the Security Threat Tool is enabled in PMM Settings, Telemetry is automatically enabled.
 >
 > - Telemetry is sent immediately; the 24-hour grace period is not honored.
+> 
+
+If you'll disable the already enabled Telemetry feature PMM will stop sending data and all feature where you can get some data from Percona will be not available to be enabled 
 
 ### Check for updates
 
 When active, PMM will automatically check for updates and put a notification in the home page *Updates* dashboard if any are available.
 
+If you'll disable the already enabled Check for Updates feature you will not be longer informed about new versions of PMM and might miss important features we'll ad in future 
```

Comment:
```suggestion
If you turn this off you will not know about important updates and features without watching our forum and release notes pages.
```

Reply:
But I'd rather omit this entirely, as our readers/users are smart and can figure it out for themselves.